packages/react-dom/src/__tests__/ReactTreeTraversal-test.js

```diff
@@ -81,10 +84,14 @@ describe('ReactTreeTraversal', () => {
     document.body.appendChild(outerNode1);
     document.body.appendChild(outerNode2);
 
-    ReactDOM.render(<ParentComponent eventHandler={mockFn} />, container);
+    root = ReactDOMClient.createRoot(container);
+    await act(() => {
+      root.render(<ParentComponent eventHandler={mockFn} />);
+    })
   });
 
   afterEach(() => {
+    root.unmount();
```

Comment:
I think this would require an act wrapper as well – or you can just leave it out that's what most other tests do as well (and this one did before).

The memory should be reclaimed after the test run anyway.